reana_job_controller/app.py

```diff
@@ -46,16 +54,110 @@ def filter_jobs(job_db):
 
 @app.route('/api/v1.0/jobs', methods=['GET'])
 def get_jobs():
+    """Get all jobs.
+
+    .. http:get:: /api/v1.0/jobs
+
+        Returns a JSON list with all the jobs.
+
+        **Request**:
+
+        .. sourcecode:: http
+
+            GET /api/v1.0/jobs HTTP/1.1
+            Content-Type: application/json
+            Host: localhost:5000
+
+        :reqheader Content-Type: application/json
+
+        **Responses**:
+
+        .. sourcecode:: http
+
+            HTTP/1.0 200 OK
+            Content-Length: 80
+            Content-Type: application/json
+
+            {
+                "status": "success",
+                "uuid": "ab19c98b-xxxx-xxxx-xxxx-1d6af3bf58b4"
+            }
+
+        :resheader Content-Type: application/json
+        :statuscode 200: no error - the list has been returned.
+    """
     return jsonify({"jobs": filter_jobs(JOB_DB)}), 200
 
 
 @app.route('/api/v1.0/k8sjobs', methods=['GET'])
 def get_k8sjobs():
+    """Get current Kubernetes job list."""
     return jsonify({"jobs": k8s.get_jobs()}), 200
 
 
 @app.route('/api/v1.0/jobs', methods=['POST'])
```

Comment:
I'm not sure we want to support several API versions in one application?! It may be better not to show version explicitly, but we can discuss this outside of the documentation context.